bigc/resources/customers.py

```diff
@@ -47,6 +47,10 @@ def create(self, first_name: str, last_name: str, email: str, **kwargs) -> dict:
         }
         return self._v3_client.request('POST', '/customers', json=[payload])[0]
 
+    def update(self, customer_id: int, data: dict) -> dict:
+        """Update a specific customer by its ID"""
+        return self._v3_client.request('PUT', f'/customers/{customer_id}', json=data)
```

Comment:
This endpoint doesn't seem to actually exist— the v3 API only has a [bulk update endpoint](https://developer.bigcommerce.com/api-reference/595425896c3ec-update-customers).